documentation/docs/06-form-actions.md

```diff
@@ -249,12 +249,14 @@ The easiest way to progressively enhance a form is to add the `use:enhance` acti
 
 Without an argument, `use:enhance` will emulate the browser-native behaviour, just without the full-page reloads. It will:
 
-- update the `form` property and invalidate all data on a successful response
-- update the `form` property on a invalid response
+- update the `form` property if the action belongs to the page and invalidate all data on a successful response
+- update the `form` property on a invalid response if the action belongs to the page
```

Comment:
I think that including the warning below is probably enough. I think it makes it harder to understand these bullet points to include more detail here
```suggestion
- update the `form` property and invalidate all data on a successful response
- update the `form` property on a invalid response
```

Reply:
You're probably right - thought the same when writing it, you confirming it makes me confident removing it again